app/src/main/java/org/gophillygo/app/data/models/Attraction.java

```diff
@@ -32,7 +27,9 @@
     private static final String LOG_LABEL = "Attraction";
 
     @PrimaryKey
-    private final int id;
+    @ColumnInfo(index = true)
+    @SerializedName("id")
+    private final int _id;
```

Comment:
Consider adding a comment explaining why we needed to switch to using `_id` instead of `id`.